src/gafaelfawr/constants.py

```diff
@@ -27,7 +27,7 @@
 CURSOR_REGEX = "^p?[0-9]+_[0-9]+$"
 """Regex matching a valid cursor."""
 
-GROUPNAME_REGEX = "^[a-z_][a-zA-Z0-9._-]*$"
+GROUPNAME_REGEX = "^[a-zA-Z0-9._-]*$"
```

Comment:
Do you have names that start with digits or capital letters that you have to use, or can we just filter them out instead? I ask because groups starting with digits are potentially going to cause a bunch of problems in downstream systems.

Reply:
yeah, we have some groups that starts with caps. but good point about the number tho

Reply:
changed in 726ecb503e5b44c89acca4a6751fa4aa70643fe2